aiocache/decorators.py

```diff
@@ -134,18 +140,21 @@ def _key_from_args(self, func, args, kwargs):
             + str(ordered_kwargs)
         )
 
-    async def get_from_cache(self, key: str):
+    async def get_from_cache(self, key):
+        namespace = self.cache.namespace
         try:
-            return await self.cache.get(key)
+            return await self.cache.get(key, namespace=namespace)
```

Comment:
Isn't the cache already using it's own namespace?

Reply:
It looks to me like the build_key method is expected to use `self.namespace` if `None`:
https://github.com/pshafer-als/aiocache/blob/decorator-namespace-key/aiocache/base.py#L499

Reply:
I don't see a single example of how to use a custom build_key, so maybe it'd make more sense if all the calls to `self.build_key()` did `namespace=namespace or self.namespace`. Then it's more likely to work as expected.

I think this is potentially backwards-incompatible, so I'd prefer if we did this change in a later PR. Can we get this working without passing the namespace in? Even if a couple of tests need to have xfail on them until that change is made.

Reply:
Yes, that was on my mind also.  Some of this is remnant of a shift in strategy to better use the underlying case, rather than implement namespace functionality directly in the decorator.

I'll clean this up and see how the test perform.